src/kibana/directives/aria_expanded.js

```diff
@@ -0,0 +1,23 @@
+define(function (require) {
+  require('modules')
+    .get('kibana')
+    .directive('ariaExpanded', function () {
+      return {
+        restrict: 'A',
+        link: function ($scope, $elem, attrs) {
+          $elem.bind('click', function () {
+            if ($elem[0].getAttribute('aria-expanded') === 'true') {
```

Comment:
The original question as I understand it was if Dom manipulation from the timepicker file could be avoided using a directive. I understood this not as recommendation but rather a request to avoid Dom manipulation from the specific file. 

From where would you like the Dom manipulation to occur?